src/openms/source/FORMAT/ExperimentalDesignFile.cpp

```diff
@@ -185,6 +179,16 @@ namespace OpenMS
       // the file section
       std::map <String, Size> fs_column_header_to_index;
 
+      // Maps the sample number to all values of sample-related columns
```

Comment:
what are sample related columns?

Reply:
columns of the "sample section" which is implicit in a one table format.